assets/js/lib/api/operations.js

```diff
@@ -1,4 +1,15 @@
-import { post } from '@lib/network';
+import { networkClient, post } from '@lib/network';
+
+// eslint-disable-next-line no-undef
+const baseURL = config.checksServiceBaseUrl;
```

Comment:
I'm reusing this value as the at the end, the checks server and operations endpoints are in the same url.
I will open a tech-debt ticket to change this, but let's better do it before the release of the feature. It will require changing many things in deployment projects and documentation